scripts/process_mturk_intent_slot_data.py

```diff
@@ -0,0 +1,156 @@
+import argparse
+import os
+import csv
+import json
+import random
+
+# Parsing arguments
+parser = argparse.ArgumentParser(
+    description='Joint intent slot filling system with pretrained BERT')
+parser.add_argument("--data_dir", default='data/mturk', type=str)
+parser.add_argument("--classification_data", default='classification.csv',
+                    type=str)
+parser.add_argument("--annotation_data", default='annotation.manifest',
+                    type=str)
+parser.add_argument("--anno_task_name", default='retail-data', type=str)
+parser.add_argument("--num_annotators", default=3, type=str)
+parser.add_argument("--dataset_name", default='mturk', type=str)
+parser.add_argument("--do_lower_case", action='store_false')
+parser.add_argument("--shuffle_data", action='store_false')
+
+args = parser.parse_args()
+
+
+def readCSV(file_path):
+    rows = []
+    with open(file_path, 'r') as csvfile:
+        readCSV = csv.reader(csvfile, delimiter=',')
+        for row in readCSV:
+            rows.append(row)
+    return rows
+
+
+def write_files(data, outfile):
+    with open(f'{outfile}', 'w') as f:
+        for item in data:
+            item = f'{item.strip()}\n'
```

Comment:
just want to make sure that you don't just remove \n token with .strip() then adding it back in